apps/dashboard/src/components/primitives/sonner-helpers.tsx

```diff
@@ -17,3 +17,20 @@ export const showToast = ({
     ...options,
   });
 };
+
+export const promoteToast = ({
```

Comment:
please use the `showToast` function instead of creating a new function; check the examples throughout the code or contact me for a help ;)

Reply:
btw please pull the latest changes ;)